File: Latihan_b/Php/Vga.php
<?php

class Vga {

    // atribut
    private $brand;
    private $type;
    private $price;


    // konstruktor
    function __construct($brand = "", $type = "", $price = 0)
    {
        $this->brand = $brand;
        $this->type = $type;
        $this->price = $price;
    }

    // setter
    function setBrand($brand)
    {
        $this->brand = $brand;
    }
        
    function setType($type)
    {
        $this->type = $type;
    }

    function setPrice($price)
    {
        $this->price = $price;
    }

    // getter
    function getBrand()
    {
        return $this->brand;
    }
        
    function getType()
    {
        return $this->type;
    }

    function getPrice()
    {
        return $this->price;
    }
}


?>

File: Latihan_b/Java/Vga.java
class Vga {

    // atribut
    private String brand;
    private String type;
    private int price;

    
    // konstruktor
    public Vga(){}

    public Vga(String brand, String type, int price)
    {
        this.brand = brand;
        this.type = type;
        this.price = price;
    }

    // setter
    public void setBrand(String brand)
    {
        this.brand = brand;
    }
        
    public void setType(String type)
    {
        this.type = type;
    }

    public void setPrice(int price)
    {
        this.price = price;
    }

    // getter
    public String getBrand()
    {
        return this.brand;
    }
        
    public String getType()
    {
        return this.type;
    }

    public int getPrice()
    {
        return this.price;
    }
}